ganga/GangaDirac/Lib/RTHandlers/DiracRTHScript.py.template

```diff
@@ -26,7 +26,7 @@ j.setParametricInputData(###PARAMETRIC_INPUTDATA###)
 ###DIRAC_OPTS###
 
 # submit the job to dirac
-j.setPlatform('###PLATFORM###')
+j.setPlatform( '###PLATFORM###' )
```

Comment:
Is this necessary as it violates PEP8?

Reply:
We can change the substitution if you prefer.
However I know this fixes the problem - I cannot guarantee I will find all the runtime handlers to change the substitution so I think this would be best.

Reply:
no stick with what works 😉